client/reader/stream/post.jsx

```diff
@@ -171,6 +171,13 @@ export default class ReaderPostCardAdapterFluxContainer extends React.Component
 		FeedPostStore.off( 'change', this.updateState );
 	}
 
+	shouldComponentUpdate( nextProps, nextState ) {
+		const shouldUpdate = this.props !== nextProps ||
```

Comment:
like `if ( this.props === nextProps || shallowEquals( this.props, nextProps ) ) { doNotUpdate }`